build/yamls/base/agent.yml

```
@@ -48,6 +48,11 @@ spec:
               add:
                 # SYS_MODULE is required to load the OVS kernel module.
                 - SYS_MODULE
+          env:
+            # SKIP_CNI_BINARIES takes in values as a comma separated list of
+            # binaries that need to be skipped for installation, e.g. "portmap, bandwdith".
```

```suggestion
            # binaries that need to be skipped for installation, e.g. "portmap, bandwidth".
```
This caused spelling check failure

---

my bad, I din't gave a relook to it. I will make the change